src/cnaas_nms/db/mgmtdomain.py

```diff
@@ -15,6 +16,8 @@
 from cnaas_nms.db.device import Device
 from cnaas_nms.db.reservedip import ReservedIP
 
+RESERVED_SKIP_COUNT = 5  # Skip this number of reserved addresses on each management domain
```

Comment:
It's still hardcoded as value 5, maybe should be configurable?

Reply:
One PR to put the magic number in a constant, and then another PR to remove the constant and put it in a config variable? I think it's fine to do in the same PR :)

Reply:
> One PR to put the magic number in a constant, and then another PR to remove the constant and put it in a config variable? I think it's fine to do in the same PR :)

Where would you have said config option?

Reply:
I guess it will go in api.yml since it has everthing that is not db, plugins or repository config :sweat_smile: 
I added a config option in a PR very recently basically these files needs updating:
docs/configuration/index.rst
src/cnaas_nms/app_settings.py
docker/api/config/api.yml

Reply:
Alright, @indy-independence  - pushed one more commit that I believe will accomplish this.